oaknational/Oak-Web-Application
# feat: 1113 update zod type to align with Data Tools

## Description

Update zod type to align with Data Tools

## Issue(s)

Fixes #1113

## How to test

1. Check if download page is working:
 https://deploy-preview-1311--oak-web-application.netlify.thenational.academy/beta/teachers/key-stages/ks2/subjects/computing/units/sharing-information-adc8/lessons/systems-cgvpad/downloads
